docs/source/physical_robot_setup/index.rst

```diff
@@ -0,0 +1,137 @@
+===================================
+Setting up a new modular robot core
+===================================
+This tutorial is for maintainers from the CI Group who want to set up a new modular robot core.
+The following tutorial does not touch the matter of building a complete robot, it just covers the setup for the **core**.
+If you simply want to know how to use a pre-installed physical robot, check the examples. This is most likely the case if you are a student.
+
+For anyone outside of the CI Group, please look at the v2 modular robot.
+
+-----------
+Preparation
+-----------
+Firstly make sure to have all the materials necessary to build the physical robot.
+For **CI-Group:**
+Models to print the core can be found on the following git repo: `<https://github.com/ci-group/revolve-models>`_.
+
+^^^^^^^^^^^^^^^^^^^^
+Install the Hardware
+^^^^^^^^^^^^^^^^^^^^
+For any type of hardware, there are some required items:
+
+* A controller such as an RPi (Raspberry Pi).
+* A hat (hardware attached on top) for the RPi to connect servos and other extensions (such as sensors or other actuators). Note that for v1 and v2 hats already exist.
+* Cables and a Battery
+
+"""""""""""""""""""""""""""""
+Hardware setup for a V1 Robot
+"""""""""""""""""""""""""""""
+**This is only applicable to the CI-Group-specific V1 design.**
+
+
+Install the RPi in the Head compartment.
+Instructions about the assembly of robots can be found in the *how-to-build-a-modular-robot-guide*.
+
+"""""""""""""""""""""""""""""
+Hardware setup for a V2 Robot
+"""""""""""""""""""""""""""""
+**This is only applicable to the CI-Group-specific V2 design.**
+
+
+Install the RPi in the Head compartment.
```

Comment:
I think we can leave this out and directly refer to the robohat git as you do below